test/models/vote_test.rb

```diff
@@ -0,0 +1,11 @@
+require "test_helper"
+
+describe Vote do
+  it "must be valid" do
```

Comment:
You should also have a validation to enforce a user & work combination has to be unique.